src/main/java/me/crespel/runtastic/RuntasticExportConverter.java

```diff
@@ -61,7 +69,25 @@ protected void doList(File path) throws FileNotFoundException, IOException {
 		SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
 		List<SportSession> sessions = converter.listSportSessions(path);
 		for (SportSession session : sessions) {
-			System.out.println(sdf.format(session.getStartTime()) + " - ID: " + session.getId() + ", type: " + session.getSportTypeId() + ", duration: " + session.getDuration() + "s");
+			System.out.println(sdf.format(session.getStartTime()) + " - ID: " + session.getId() + ", Sport Type: " + session.getSportTypeId() + ", duration: " + session.getDuration()/60000 + "min");
+		}
+	}
+
+	protected void doInfo(File path, String id) throws FileNotFoundException, IOException {
+		SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+		List<SportSession> sessions = converter.listSportSessions(path);
+		for (SportSession session : sessions) {
+			if ( session.getId().equalsIgnoreCase(id) ) {
+				System.out.println(sdf.format(session.getStartTime()) + " - ID: " + session.getId() );
+				System.out.println("      Sport Type: " + session.getSportTypeId() + ", Surface Type: " + session.getSurfaceId() + ", Feeling Id: " + session.getSubjectiveFeelingId());
+				System.out.println("      Duration: " + session.getDuration() + " ms / " + session.getDuration()/1000 + " s / " + session.getDuration()/60000 + " min");

```

Comment:
Possible improvement: display the duration as hours:minutes:seconds for easier reading.

Reply:
Removed the [ms] and [s] values; I think showing [min] is good enough.

Reply:
Will be fixed in next commit.